src/main/features/claim/routes/claimant-add.ts

```diff
@@ -51,6 +51,8 @@ export default express.Router()
           Claimants.addClaimant(res)
           await ClaimDraftMiddleware.save(res, next)
           res.redirect(Paths.claimantTypePage.uri)
+        } else if (res.locals.user.legalClaimDraft.defendants.length > 1) {
```

Comment:
what is this step for?? is it requirement??

Reply:
If we are coming from the details summary page, and we have multiple defendants, then the next page should be defendant-add where we have the table of defendants.

Reply:
or should have displayed last defendant details?? was this requirement?

Reply:
displaying last defendant and then going to defendant table does not make any sense.